components/src/core/components/List/types.ts

```diff
@@ -0,0 +1,126 @@
+interface Visible {
+  visible?: boolean;
+}
```

Comment:
I'm not sure if `Visible` is needed. It's basically an alias for boolean.
Is this a standard used in typescript?

Reply:
No, there's no standard for this but "visibile" is getting referred in many places again and again therefore I added an interface for that and extended